### PR TITLE
adding drawer

### PR DESCRIPTION
**Pull request by kr1skyyy**

### CODE REVIEW

File: frontend/src/components/app/App.tsx [hunk 1]
```diff
@@ -1,31 +1,17 @@
...
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
-      </div>
-    </ApolloProvider>
+    //<ApolloProvider client={client}>
```
Comment (kr1skyyy) [inline frontend/src/components/app/App.tsx:12]: Why did you comment the apollo provider?

File: frontend/src/components/app/sidebar/drawer.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    return (
+        <Router>
+            <div style={{ display: 'flex' }}>
+                <Drawer
+                    style={{ width: '220px' }}
+                    variant='persistent'
+                    anchor='left'
+                    open={true}
+                    classes={{ paper: classes.drawerPaper }}>
+
```
Comment (kr1skyyy) [inline frontend/src/components/app/sidebar/drawer.tsx:25]: unnecessary empty line

File: frontend/src/components/app/sidebar/drawer.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                </Switch>
+
+            </div>
+        </Router>
+    );
+}
+
+export default CreateDrawer;
+
+
```
Comment (kr1skyyy) [inline frontend/src/components/app/sidebar/drawer.tsx:80]: same as above, only one new line

File: frontend/src/components/app/App.tsx (hunk 1 above)
Comment (ilko1999) [reply frontend/src/components/app/App.tsx:12]: it was giving me import warning on the page when rendered

**Pull request merged**